.github/labeler.yml

```diff
@@ -17,3 +17,33 @@ documentation:
   - docs/**/*
   - CONTRIBUTING.rst
   - README.rst
+
+component/weaver:
+  - birdhouse/components/weaver/**/*
+  - birdhouse/**/weaver/**/*
+
+component/cowbird:
+  - birdhouse/components/cowbird/**/*
+  - birdhouse/**/cowbird/**/*
+
+component/magpie:
+  - birdhouse/config/magpie/**/*
+  - birdhouse/**/magpie/**/*
```

Comment:
In anticipation of the big move, are we allowed to proactively add the future path `birdhouse/components/magpie/**/*`?  Same comments for all other components.

Another question is `birdhouse/**/magpie/**/*` already include everything before so do we even need to specify `/config|components/magpie`?

Reply:
Yes. We can add them too, but it shouldn't be necessary as mentioned by the globs including everything.

Reply:
Won't these globs match directories like `birdhouse/config/finch/config/magpie/` as well? Is that desirable here?

Reply:
Indeed. I think this is valid. It will apply both tags for magpie/wps, which will be relevant if a file in there was modified.